examples/common/utils_examples.h

```diff
@@ -0,0 +1,16 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#ifndef UTILS_EXAMPLES_H
+#define UTILS_EXAMPLES_H
+
+// Needed for CI
+#define test_skip_error_code 125
```

Comment:
Defines are usually in CAPITAL letters. Could you replace it?

Reply:
Done